src/javascript/app/pages/trade/process.js

```diff
@@ -242,6 +243,7 @@ const Process = (() => {
             } else {
                 Defaults.set('selected_tick', selected_tick.value);
             }
+            Reset.hideResetTime();
```

Comment:
why specifically for this type of trade? can't we hide it in a more general place?